### DISCUSSION TITLE
problem in output.css

### DISCUSSION THREAD

**Discussion by Aishivam1** [tailwindlabs/tailwindcss#14131] [Help]
when i run command and it generate style .css  file but it always blank and terminal show--
![Screenshot 2024-08-06 102319](https://github.com/user-attachments/assets/36de8178-15c3-4c1f-891f-c9875fe1ed8f)

Rebuilding...

Done in 278ms.

**Comment by philipp-spiess** [comment]
@Aishivam1 Hey! Can you share a reproduction case (e.g. a GitHub repo that you use to get this issue)?
